File: python/8-agents/after-python/StarWarsCopilot/mcp_server_options.py
from pathlib import Path


class MCPServerOptions:
    @staticmethod
    def name() -> str:
        return "StarWarsMCPServer"

    @staticmethod
    def command() -> str:
        return "python"

    @staticmethod
    def arguments() -> list[str]:
        server_path = (
            Path(__file__).resolve().parents[3]
            / "7-multimodal"
            / "after-python"
            / "StarWarsMCPServer"
            / "server.py"
        )
        return [str(server_path)]

File: csharp/7-multimodal/after/StarWarsCopilot/MCPServerOptions.cs
namespace StarWarsCopilot;

public static class MCPServerOptions
{
    public static string Name => "StarWarsMCPServer";
    public static string Command => "dotnet";
    public static List<string> Arguments => [
        "run",
        "--project",
        "<path>/StarWarsMCPServer.csproj"
    ];
}